src/components/gameAnswers/gameAnswers.js

```diff
@@ -0,0 +1,42 @@
+import './gameAnswers.css';
+import isAnswerCorrect from '../../services/game/isAnswerCorrect';
+
+export function gameAnswers(answers, rightAnswer, callback) {
```

Comment:
`rightAnswer` -> `correctAnswer`

Reply:
`gameAnswers` name isn't say what excatly this component does
could you rethink name, I think it could be something like `answersModal` or `answersCard`